modules/agent-host-roles.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="agent-host-roles_{context}"]
+= Host roles
+
+Each host in the cluster is assigned a role of either `master` or `worker`.
+You can define the role for each host in the `agent-config.yaml` file by using the `role` parameter.
+If you do not assign a role to the hosts, the roles will be assigned at random during installation.
+
+It is recommended to explicitly define roles for your hosts.
+
+The `rendezvousIP` must be assigned to a host with the `master` role, whether you explicitly assign the host role or whether the Agent-based Installer assigns the role.
+
+[IMPORTANT]
+====
+You do not need to explicitly define the `master` role for the rendezvous host, however you cannot create configurations that conflict with this assignment.
+
+For example, if you have 4 hosts with 3 explicitly defined to have the `master` role, then the last host, which will automatically be assigned the `worker` role during installation, cannot be configured as the rendezvous host.
```

Comment:
For example, if you have 4 hosts with 3 of the hosts explicitly defined to have the `master` role, the last host that is assigned the `worker` role during installation cannot be configured as the rendezvous host.

Reply:
I think this suggestion sounds a little better, thanks